src/components/svg-wrapper.tsx

```diff
@@ -72,6 +73,12 @@ const SvgWrapper = () => {
             // only type is needed
             .forEach(([type]) => loadFontCss(type as NodeType));
     }, [refreshNodes]);
+    // Update all components that requires a bbox after fonts are loaded.
+    // bbox calculated before fonts are loaded will be incorrect.
+    const { update } = React.useContext(SvgAssetsContext);
```

Comment:
Does this line work outside of the context provider?

Reply:
Moved to src/components/svgs/stations/gzmtr-int-2024.tsx